tools/pylint-extensions/azure-pylint-guidelines-checker/tests/test_pylint_custom_plugins.py

```diff
@@ -4845,55 +4846,6 @@ def test_allowed_import_else(self):
             self.checker.visit_importfrom(imd)
 
 
-
-class TestDoNotImportAsyncio(pylint.testutils.CheckerTestCase):
```

Comment:
It was throwing an error because there was no corresponding class in the checker file. i had done a merge incorrectly and this test wasnt supposed to be in the "Error if same named type importable" branch

Reply:
Ah I see! It should be merged into main now if you merge main hopefully that will sort it out :)